include/openssl/evp.h

```diff
@@ -141,6 +140,10 @@ OPENSSL_EXPORT int EVP_PKEY_id(const EVP_PKEY *pkey);
 // otherwise.
 OPENSSL_EXPORT int EVP_PKEY_type(int nid);
 
+// EVP_MD_get0_name returns the short name of |md|
+OPENSSL_EXPORT const char *EVP_MD_get0_name(const EVP_MD *md);
+#define EVP_MD_name EVP_MD_get0_name
```

Comment:
Is there a particular way that NTP is consuming this as a macro?  We could define `EVP_MD_name`  and `EVP_MD_pkey_type` as functions anyways and add them to this section below?
https://github.com/aws/aws-lc/blob/95cc2ce94a00304eaa887a80b851b89e5f0c65d8/include/openssl/evp.h#L1158-L1169

Reply:
Not sure what you mean, NTP is just calling `name = EVP_MD_name(m);` in ntpq.c. They are using it in a way that this could be a function. But in theory other consumers could try to use it as a macro in `if defined(EVP_MD_name)`.

Reply:
The section I linked to redefines the functions as macros, so consumers can still detect it with `#if defined(EVP_MD_name)`. The comment within the section also specifies that this was created to avoid breaking `#idefs` in consumers: https://github.com/aws/aws-lc/blob/95cc2ce94a00304eaa887a80b851b89e5f0c65d8/include/openssl/evp.h#L1160-L1162

I think we could still define `EVP_MD_name` and `EVP_MD_pkey_type` as functions, and add them to the `Preprocessor compatibility section`. It's a consistent trend we've been doing, `ssl.h` has more examples here:
https://github.com/aws/aws-lc/blob/95cc2ce94a00304eaa887a80b851b89e5f0c65d8/include/openssl/ssl.h#L5657-L5803